omod/src/main/java/org/openmrs/module/coreapps/helper/CurrentActiveVisit.java

```diff
@@ -0,0 +1,46 @@
+package org.openmrs.module.coreapps.helper;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.Patient;
+import org.openmrs.Visit;
+import org.openmrs.api.VisitService;
+import org.openmrs.api.context.Context;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+/**
+ * This class is used to get current(latest) active visit for a specific patient.
+ */
+public class CurrentActiveVisit {
```

Comment:
You don't need to write this logic, you should use the existing logic. See here:
https://github.com/openmrs/openmrs-module-coreapps/blob/master/omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/PatientHeaderFragmentController.java#L75-L76